python/ray/tune/syncer.py

```diff
@@ -59,31 +59,68 @@ class SyncConfig:
 
     If an ``upload_dir`` is specified, both experiment and trial checkpoints
     will be stored on remote (cloud) storage. Synchronization then only
-    happens via this remote storage.
+    happens via uploading/downloading from this remote storage - no syncing will
```

Comment:
@amogkam Do these docstring updates clarify some of the confusions you had when you tried learning more about Tune syncing?